apps/condo/lang/ru/messages/DIRTY_INVITE_NEW_EMPLOYEE_EMAIL/default.njk

```diff
@@ -1 +1 @@
-"{{ message.meta.organizationName }}" приглашает в Дома: {{ message.meta.serverUrl }}
+"{{ message.meta.organizationName }}" приглашает в Дома: {{ message.meta.serverUrl }}/auth/{% if message.meta.isRegistration %}register{% else %}signin{% endif %}
```

Comment:
It seems that this text message is being sent to already registered users.
Is it ok to send message to /auth if user already logged in?

Reply:
The user may be registered, but without a password he will not be able to log in, so we send him to complete the registration.

If the user already exists, then we open the login page

Reply:
updated